desktop/src/main/java/bisq/desktop/main/dao/governance/result/VoteResultView.java

```diff
@@ -238,7 +239,11 @@ protected void deactivate() {
 
     @Override
     public void onParseBlockCompleteAfterBatchProcessing(Block block) {
-        int chainHeight = daoStateService.getChainHeight();
+        checkForResultPhase(daoStateService.getChainHeight());
+        fillCycleList();
+    }
+
+    private void checkForResultPhase(int chainHeight) {
         if (periodService.getFirstBlockOfPhase(chainHeight, DaoPhase.Phase.RESULT) == chainHeight) {
```

Comment:
Shouldn't this be `<= chainHeight`?

Reply:
`periodService.getFirstBlockOfPhase` gets the height of the first block of the current phase so this check will return true for all `chainHeight` in the `RESULT` phase.